lib/firebase_storage.js

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
If that's the case, would you please either submit a pull request or an issue? Thanks!